Gemfile

```diff
@@ -34,8 +35,9 @@ group :development do
   gem 'web-console', '>= 3.3.0'
   gem 'listen', '~> 3.2'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem 'spring'
-  gem 'spring-watcher-listen', '~> 2.0.0'
+  # ...fixes failure of rails generate migration....
+  gem 'spring', '~> 4.1'
+  # gem 'spring-watcher-listen', '~> 2.0.0'
```

Comment:
I'd like to avoid adding comments when we don't need to. Can we remove the new comment on line 38? I think it's a good habit to learn any details directly from the github page for the gem.

And I'm ok with commenting out the `spring-watcher-listen` gem if we think we'll need it in the future, otherwise let's just remove the line. Was there a particular reason you're removing this one?